views_lnurl.py

```diff
@@ -137,32 +138,23 @@ async def lnurl_callback(
     if fossa_payment.payload != k1:
         await delete_atm_payment_link(payment_id)
         return {"status": "ERROR", "reason": "Bad K1"}
-    if fossa_payment.payhash != "payment_hash":
+    if fossa_payment.payment_hash != "payment_hash":
         return {"status": "ERROR", "reason": "Payment already claimed"}
     try:
-        fossa_payment.payhash = "pending"
-        fossa_payment_updated = await update_fossa_payment(
-            CreateFossaPayment(**fossa_payment.dict(exclude={"timestamp"}))
-        )
-        assert fossa_payment_updated
+        fossa_payment.payment_hash = "pending"
+        fossa_payment_updated = await update_fossa_payment(fossa_payment)
         await pay_invoice(
             wallet_id=device.wallet,
             payment_request=pr,
             max_sat=int(fossa_payment_updated.sats) + 100,
             extra={"tag": "fossa_withdraw"},
         )
-        fossa_payment.payhash = fossa_payment.payload
-        fossa_payment_updated = await update_fossa_payment(
-            CreateFossaPayment(**fossa_payment.dict(exclude={"timestamp"}))
-        )
-        assert fossa_payment_updated
+        fossa_payment.payment_hash = fossa_payment.payload
+        fossa_payment_updated = await update_fossa_payment(fossa_payment)
         return {"status": "OK"}
     except HTTPException as e:
         return {"status": "ERROR", "reason": str(e)}
     except Exception as e:
-        fossa_payment.payhash = "payment_hash"
-        fossa_payment_updated = await update_fossa_payment(
-            CreateFossaPayment(**fossa_payment.dict(exclude={"timestamp"}))
-        )
-        assert fossa_payment_updated
+        fossa_payment.payment_hash = "payment_hash"
```

Comment:
should this be `Optional` like in `lnpos` instead of having `"payment_hash"` hardcoded